docker-setup/snowflake-kafka-docker/m1-kafka-connect.yml

```diff
@@ -0,0 +1,139 @@
+      SCHEMA_REGISTRY_KAFKASTORE_BOOTSTRAP_SERVERS: 'PLAINTEXT://kafka:29092,PLAINTEXT://localhost:9092'
+      
+  connect:
+    image: confluentinc/cp-kafka-connect:latest
+    platform: linux/arm64
+    hostname: connect
+    container_name: kafka-connector
+    depends_on:
+      - kafka
+    ports:
+      - "8083:8083"
+      - 42514:42514/udp
+      - 5060:5060 # KAFKA_DEBUG port
+      - 9876:9876 # JMX port
+      - 8085:8085
+    volumes:
+      - /usr/local/share/kafka/plugins:/etc/kafka-connect/jars/ # if you want your custom jar, put custom jar in ./jars directory
+    environment:
+      CONNECT_BOOTSTRAP_SERVERS: 'kafka:29092'
+      CONNECT_REST_ADVERTISED_HOST_NAME: connect
+      CONNECT_REST_PORT: 8083
+      CONNECT_GROUP_ID: compose-connect-group
+      CONNECT_CONFIG_STORAGE_TOPIC: docker-connect-configs
+      CONNECT_CONFIG_STORAGE_REPLICATION_FACTOR: 1
+      CONNECT_OFFSET_FLUSH_INTERVAL_MS: 10000
+      CONNECT_OFFSET_STORAGE_TOPIC: docker-connect-offsets
+      CONNECT_OFFSET_STORAGE_REPLICATION_FACTOR: 1
+      CONNECT_STATUS_STORAGE_TOPIC: docker-connect-status
+      CONNECT_STATUS_STORAGE_REPLICATION_FACTOR: 1
+      CONNECT_KEY_CONVERTER: org.apache.kafka.connect.storage.StringConverter
+      CONNECT_VALUE_CONVERTER: org.apache.kafka.connect.json.JsonConverter
+      CONNECT_INTERNAL_KEY_CONVERTER: "org.apache.kafka.connect.json.JsonConverter"
+      CONNECT_INTERNAL_VALUE_CONVERTER: "org.apache.kafka.connect.json.JsonConverter"
+      # CLASSPATH required due to CC-2422
+      CLASSPATH: /usr/share/java/monitoring-interceptors/monitoring-interceptors-7.1.0.jar
+      CONNECT_PLUGIN_PATH: "/usr/share/java,/usr/share/confluent-hub-components,/etc/kafka-connect/jars"
+      JMX_PORT: 9876
+      CONNECT_LOG4J_LOGGERS: org.apache.zookeeper=ERROR,org.I0Itec.zkclient=ERROR,org.reflections=ERROR,com.snowflake=DEBUG
+      KAFKA_DEBUG: true
+      JAVA_DEBUG_PORT: 0.0.0.0:5060
+    command:
+      - bash
+      - -c
+      - |
+        echo "Installing Snowflake kafka connector"
+        # if jar is available on confluent hub, uncomment below line
+        confluent-hub install --no-prompt snowflakeinc/snowflake-kafka-connector:1.8.1
+        echo "Launching Kafka Connect worker"
+        /etc/confluent/docker/run
+
+  control-center:
+    image: confluentinc/cp-enterprise-control-center:latest
+    platform: linux/arm64
+    hostname: control-center
+    container_name: kafka-connect-control-center
+    depends_on:
+      - kafka
+      - connect
+    ports:
+      - "9021:9021"
+    environment:
+      CONTROL_CENTER_BOOTSTRAP_SERVERS: 'kafka:29092'
+      CONTROL_CENTER_ZOOKEEPER_CONNECT: 'zookeeper:2181'
+      CONTROL_CENTER_CONNECT_CLUSTER: 'connect:8083'
+      CONTROL_CENTER_REPLICATION_FACTOR: 1
+      CONTROL_CENTER_INTERNAL_TOPICS_PARTITIONS: 1
+      CONTROL_CENTER_CONNECT_HEALTHCHECK_ENDPOINT: '/connectors'
+      #CONTROL_CENTER_LOG4J_ROOT_LOGLEVEL: "DEBUG" # uncomment for debug logs
+      PORT: 9021
```

Comment:
Should we bring up a discussion in confluent sync that control center is not working on m1 machines?

Reply:
I mentioned it during the kafka connector sync and raised an issue on the confluent control center github: https://github.com/confluentinc/control-center-images/issues/55